analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ProjectConfigReaderTests.cs

```diff
@@ -88,9 +82,7 @@ public void WhenHasMissingFilesToAnalyzePath_ReturnsCorrectValue()
         [TestMethod]
         public void WhenHasUnexpectedProjectType_FallsBackToProduct()
         {
-            var options = TestHelper.CreateOptions($"ResourceTests\\SonarProjectConfig\\UnexpectedProjectTypeValue\\SonarProjectConfig.xml");
-
-            var sut = new ProjectConfigReader(options);
+            var sut = CreateProjectConfigReader($"ResourceTests\\SonarProjectConfig\\UnexpectedProjectTypeValue\\SonarProjectConfig.xml");
```

Comment:
Wherever the variable is used only once I am not sure it is needed. From the class name ProjectConfigReaderTests it is clear that the sut is ProjectConfigReader